src/uk/gov/hmcts/contino/GradleBuilder.groovy

```diff
@@ -50,8 +52,10 @@ class GradleBuilder extends AbstractBuilder {
       // By default Gradle will skip task execution if it's already been run (is 'up to date').
       // --rerun-tasks ensures that subsequent calls to tests against different slots are executed.
       gradle("--rerun-tasks smoke")
+    } catch(ignored) {
+      WarningCollector.addPipelineWarning("deprecated_smoke_test_archiving", "Please setup smoke tests to archive according to the correct pattern. See <add in example PR here>", new Date().parse("dd.MM.yyyy", "11.05.2022"))
     } finally {
-      steps.junit '**/test-results/**/*.xml'
+      steps.junit 'smoke-output/**/*result.xml'
```

Comment:
should be something like (but needs checking with a few projects, some prob use smoke, some may use smokeTest, some may be functional):
```suggestion
      steps.junit '**/test-results/smoke/*.xml'
```